app/src/main/java/com/kms/cura/model/request/PatientModelResponse.java

```diff
@@ -0,0 +1,68 @@
+package com.kms.cura.model.request;
+
+import com.android.volley.VolleyError;
+import com.kms.cura.constant.EventConstant;
+import com.kms.cura.entity.json.JsonToEntityConverter;
+import com.kms.cura.entity.user.PatientUserEntity;
+import com.kms.cura.entity.user.UserEntity;
+import com.kms.cura.event.EventBroker;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Created by toanbnguyen on 7/14/2016.
+ */
+public class PatientModelResponse implements EntityModelResponse {
+    PatientUserEntity entity;
+    boolean gotResponse;
+    boolean responseError = false;
+    String error;
+
+    public PatientModelResponse() {
+    }
+
+    @Override
+    public void onResponse(String response) {
+        JSONObject jsonObject;
+        try {
+            jsonObject = new JSONObject(response);
+            boolean status = jsonObject.getBoolean(UserEntity.STATUS_KEY);
+            if (status) {
+                entity = JsonToEntityConverter.convertJsonStringToEntity(response, PatientUserEntity.getPatientUserType());
+                EventBroker.getInstance().pusblish(EventConstant.UPDATE_SUCCESS, EventConstant.TYPE_PATIENT);
```

Comment:
After using the specific event UPDATE_PATIENT_HEALTH_SUCCESS , pass the PatientUserEntity as data for the event to whoever listening